android-components/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -294,6 +295,10 @@ object WebExtensionSupport {
                     store.dispatch(WebExtensionAction.UninstallAllWebExtensionsAction)
                     registerInstalledExtensions(store, runtime)
                 }
+
+                override fun onDisabledExtensionProcessSpawning() {
+                    onDisabledExtensionProcessSpawning?.invoke()
```

Comment:
I think we could add a this on `WebExtensionSupportTest` to verify this behaviour :)

Reply:
agreed, will add